lib/routes/auth/resetPassword.js

```diff
@@ -15,9 +13,9 @@ module.exports = {
             const AuthService = h.services().authService;
             const { oldPassword, userHash, forgotPasswordToken, newPassword } = request.payload;
 
-            const results = await AuthService.resetPassword(request.auth.credentials, oldPassword, userHash, forgotPasswordToken, newPassword);
+            await AuthService.resetPassword(request.auth.credentials, oldPassword, userHash, forgotPasswordToken, newPassword);
 
-            return new SuccessResult(results);
+            return h.response().code(204);
         }
```

Comment:
If I understand correctly, if I need the data returned from calling `AuthService.resetPassword`, I'll use `/resetpassword2` route. If I don't and I only want perform the action of resetting the password, I'll use this one?

Reply:
I think this is just some refactoring slop. `TokenService.resetPassword` doesn't return anything, so nothing would ever get return from there anyway. Just committed a change to clean that up :+1:

Reply:
Ohh got it. Thanks for clarifying 🙏